### PR TITLE
[#9034] Turn off autosuggestion on seedphrase confirmation input

### LINKED ISSUE #9034: Turn off autosuggestion on seedphrase confirmation input
***Type***: Bug

***Summary***: We have enabled keyboard autosuggestion for the input when confirming seed phrase during backup. Similar to passphrase input we have to disable  autosuggestion when user type words to confirm seed phrase remembered.

#### Expected behavior

No keyboard autosuggestion on seedphrase confirmation input

#### Actual behavior
Keyboard autosuggestion on seedphrase confirmation input is active

Screenshot 2019-09-25 at 12 41 14

### Reproduction

- Open Status and create new account
- Navigate to `Profile` -> `Privacy and security` -> `Back up seed phrase` -> tap `Next`
- Start typing in the input field

### Additional Information
- Status version: develop `0.14.0 (2019092402)`
- Operating System: Android and iOS

### PR DESCRIPTION
**Pull request by flexsurfer**
fixes #9034

### CODE REVIEW

File: src/status_im/ui/components/text_input/view.cljs [hunk 1]
```diff
@@ -14,7 +14,7 @@
       (dissoc merged-styles :background-color)
       merged-styles)))
 
-(defn text-input-with-label [{:keys [label content error style height container text editable] :as props :or {editable true}}]
+(defn text-input-with-label [{:keys [label content error style height container text editable keyboard-type] :as props :or {editable true}}]
```
Comment (yenda) [inline src/status_im/ui/components/text_input/view.cljs]:
```suggestion
(defn text-input-with-label 
  [{:keys [label content error style height container text editable keyboard-type] 
    :as props
    :or {editable true}}]
```

**Review by vitvly — APPROVED** [review]

**Review by dmitryn — APPROVED** [review]

File: src/status_im/ui/components/text_input/view.cljs (hunk 1 above)
Comment (yenda) [reply src/status_im/ui/components/text_input/view.cljs]: please break the line it is too long

**Comment by yenda** [comment]
@flexsurfer please include my suggestion in the commit.

Also I tested and the suggestion is gone in the profile but they are still there when you are creating an account with keycard and you are asked to confirm seed phrase

**Comment by yenda** [comment]
why you merged the suggestion are still there in onboarding :(

**Comment by flexsurfer** [comment]
@yenda my bad, hadn't seen your comment :(

**Comment by flexsurfer** [comment]
https://github.com/status-im/status-react/pull/9069

**Pull request merged**